lib/index.js

```diff
@@ -199,6 +200,17 @@ exports.Socket = function (type) {
     }
   }
 
+  if (this.type == 'pull') {
+    this._pause = false;
```

Comment:
@ronkorving you must know something about V8 classes that eludes me.

averaging some results here, the performance diff is not more than several nanoseconds apart.